pkg/cmd/create/cluster/scenario/create/hub/aws/install_config.yaml

```diff
@@ -24,11 +34,21 @@ compute:
   platform:
     aws:
 {{ if .managedCluster.aws.worker }}
-      rootVolume:
 {{ if .managedCluster.aws.worker.rootVolume }}
+      rootVolume:
 {{ toYaml .managedCluster.aws.worker.rootVolume | indent 8}}
-{{end}}
-{{end}}
+{{ else }}
```

Comment:
Are there supposed to be 2 else's here?

Reply:
yes as I didn't find a way to make an "and" in the "if" because each expression in the "if" are evaluate and so if the ".managedCluster.aws.worker" is nill, an panic error is generated when evaluating ".managedCluster.aws.worker.rootVolume"

Reply:
I should have looked at the whole file as it would have been easier to read...especially without indentation.